packages/playground/package.json

```diff
@@ -74,15 +74,15 @@
   "dependencies": {
     "@fluentui/react-components": "~9.60.0",
     "@fluentui/react-icons": "^2.0.279",
-    "@typespec/bundler": "workspace:~",
-    "@typespec/compiler": "workspace:~",
-    "@typespec/html-program-viewer": "workspace:~",
-    "@typespec/http": "workspace:~",
-    "@typespec/openapi": "workspace:~",
-    "@typespec/openapi3": "workspace:~",
-    "@typespec/protobuf": "workspace:~",
-    "@typespec/rest": "workspace:~",
-    "@typespec/versioning": "workspace:~",
+    "@typespec/bundler": "workspace:^",
+    "@typespec/compiler": "workspace:^",
+    "@typespec/html-program-viewer": "workspace:^",
```

Comment:
i think we should just update for the ones that will be 1.0, though I guess it doesn't matter for 0. too as they should behave the same

Reply:
yeah then I guess its fine, could even merge that now